docs/content/operations/api-reference.md

```diff
@@ -0,0 +1,426 @@
+---
+layout: doc_page
+---
+
+# API Reference
+
+This page documents all of the API endpoints for each Druid service type.
+
+## Table of Contents
+  * [Coordinator](#coordinator)
+  * [Overlord](#overlord)
+  * [MiddleManager](#middlemanager)
+  * [Peon](#peon)
+  * [Broker](#broker)
+  * [Historical](#historical)
+
+## Common
+
+The following endpoints are supported by all nodes.
+
+### GET
+
+* `/status`
+
+Returns the Druid version, loaded extensions, memory used, total memory and other useful information about the node.
+
+* `/health`
+
+An endpoint that always returns a boolean "true" value with a 200 OK response, useful for automated health checks.
+
+* `/properties`
+
+Returns the current configuration properties of the node.
+
+## Coordinator
+
+HTTP Endpoints
```

Comment:
I don't think this is necessary, it's somewhat implied and inconsistent with the other sections.

Reply:
Removed these parts